pkg/k8sapi/discovery.go

```diff
@@ -256,7 +254,7 @@ func (d *Controller) handlePodUpdate(key string) error {
 		d.workerPodsLock.Lock()
 		defer d.workerPodsLock.Unlock()
 
-		log.Tracef("Update for pod %s: %+v, %+v", podName, pod.Status, pod.Spec)
+		log.Debugf("Update for pod %s: %+v, %+v", podName, pod.Status, pod.Spec)
```

Comment:
I think this might become too verbose, since it is printing the full `pod.Spec` on every pod update on the node. The info level log on line 267 is probably enough.

Reply:
I would like to leave it as is for now as this will provide us some better info while troubleshooting.